### ISSUE TITLE
Project path is no longer resolved in SublimeLinter 3.10.0 / 3.10.1

### ISSUE THREAD

**Issue by rholliger** [SublimeLinter/SublimeLinter3#643]
I'm using a custom eslint rules file, which I get by defining a `--rulesdir /path/to/file` argument in the .sublimelinterrc file.

I have the following configurations:

**Project path:**
`/Users/username/Development/myproject`

**.sublime-project:**

```
{
  "folders":
  [
    {
      "folder_exclude_patterns":
      [
         ".happypack",
         "dist",
         "node_modules",
         "test/unit/coverage"
      ],
      "path": "." // <-- my root project directory
    }
  ]
}
```

**.sublimelinterrc:**
```
{
  "linters": {
    "eslint": {
      "args": [
        "--rulesdir",
          "${project}/eslint/rules"
      ]
    },
    ...
  }
}

```
**Expected Behavior:**

1. The ${project} token should look at the path in the .sublime-project configuration
2. The path string is ".", which is the same as the project path: `/Users/username/Development/myproject`
3. Therefore the `--rulesdir` argument should resolve to:

`/Users/username/Development/myproject/eslint/rules`

**Current Behavior:**
The `--rulesdir` argument resolves to:

`./eslint/rules`

I would assume, that this is a new bug introduced in one of the newer versions of SublimeLinter3 **(3.10.0 or 3.10.1)**? In the previous version I used **(3.9.0)**, the problem didn't exist.

**Comment by rholliger** [comment]
Maybe this is related to this change (#637)

**Comment by Palidino** [comment]
${project} is also no longer working for me for Java, is this a bug or is there a workaround/change for how to go about specifying the project directory?

**Comment by groteworld** [comment]
duplicate of #641, this should be fixed in v3.10.2
